taxi/models.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=100, unique=True)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=100)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                     related_name="drivers")
```

Comment:
Remember to add `verbose_name` and `verbose_name_plural` in lowercase in the `Meta` class for each model. This is part of the code style guidelines in the checklist.